### ISSUE TITLE
Installation using `go get` does not work

### ISSUE THREAD

**Issue by Samyak2** [mr-karan/doggo#28]
## The issue

The README instructs to use the following command to install doggo through `go get`.
```
go get github.com/mr-karan/doggo/cmd/doggo
```

But it fails with this error:
```
go get: module github.com/mr-karan/doggo@upgrade found (v0.4.0), but does not contain package github.com/mr-karan/doggo/cmd/doggo
```

## Possible cause

Looks like the CLI exists in `cmd/doggo/cli` for the `0.4.0` release - [source](https://github.com/mr-karan/doggo/tree/8dc23ce8f584e6c50e2c6442b206ed44731aec0c/cmd/doggo/cli).

## Solution

 - Use `go get github.com/mr-karan/doggo/cmd/doggo/cli`
    - This causes the binary to be named `cli` instead of `doggo`
 - Make a new release?
    - Since it looks like the paths are fixed on main

**Comment by stoiven** [comment]
+1 on having this issue as

**Comment by mr-karan** [comment]
Thanks for reporting. This should be fixed with [0.4.1](https://github.com/mr-karan/doggo/releases/tag/v0.4.1)
